lib/vmpooler/providers/vsphere.rb

```diff
@@ -422,8 +434,9 @@ def add_disk(vm, size, datastore, connection)
           true
         end
 
-        def find_datastore(datastorename, connection)
-          datacenter = connection.serviceInstance.find_datacenter
```

Comment:
Would this one need changed too?
https://github.com/glennsarti/vmpooler/blob/55d286f8bb3b60c9798546b231096ea6d0005854/lib/vmpooler/providers/vsphere.rb#L427